webui/react/src/components/kit/Breadcrumb.tsx

```diff
@@ -0,0 +1,36 @@
+import { Breadcrumb as AntdBreadcrumb } from 'antd';
+import React, { ReactNode } from 'react';
+
+interface BreadcrumbProps {
+  children?: ReactNode;
+  className?: string;
```

Comment:
The two properties we absolutely shouldn't be exposing on UI kit components are `className` and `style`.